be/src/olap/rowset/segment_v2/binary_dict_page.cpp

```diff
@@ -266,8 +267,9 @@ Status BinaryDictPageDecoder::next_batch(size_t* n, vectorized::MutableColumnPtr
         
         // fill null bitmap here, not null;
         // todo(wb) using SIMD speed up here
+        auto& null_map_data = nullable_column->get_null_map_data();
         for (int i = 0; i < max_fetch; i++) {
-            nullable_column->get_null_map_data().push_back(0);
+            null_map_data.push_back(0);
```

Comment:
the TODO batch insert has been solved by #7836

Reply:
ok, I removed this change.